include/nbl/asset/material_compiler/CMaterialCompilerGLSLBackendCommon.h

```diff
@@ -247,12 +247,11 @@ class CMaterialCompilerGLSLBackendCommon
 		using VTID = asset::ICPUVirtualTexture::SMasterTextureData;
 	#include "nbl/nblpack.h"
 		struct STextureData {
-			STextureData() = default;
 
-			VTID vtid = VTID::invalid();
+			VTID vtid;
 			union {
 				uint32_t prefetch_reg;//uint32
-				uint32_t scale = 0u;//float
+				uint32_t scale;//float
```

Comment:
removing initializers will fuck it up... now you need to make a proper constructor for this struct.